kolibri/plugins/coach/assets/src/views/class-selector/index.vue

```diff
@@ -53,42 +53,35 @@
       classOptions() {
         return this.sortedClasses.map(classroom => ({
           label: classroom.name,
-          id: classroom.id,
+          value: classroom.id,
         }));
       },
       currentClass() {
-        return this.classOptions.find(classroom => classroom.id === this.currentClassId);
+        return this.classOptions.find(classroom => classroom.value === this.currentClassId);
       },
       classListPage() {
         return { name: PageNames.CLASS_LIST };
       },
     },
     methods: {
       changeClass(classSelected) {
-        this.$emit('changeClass', classSelected.id);
+        this.$emit('changeClass', classSelected.value);
       },
     },
   };
 
 </script>
 
 
-<style lang="stylus">
-
-  .class-selector
-    .ui-select__display-value
-      font-size: 1.5em
-      font-weight: bold
-
-</style>
-
-
 <style lang="stylus" scoped>
 
   .class-selector
-    display: inline-flex
     vertical-align: bottom
 
+  >>>.ui-select__display-value
```

Comment:
Git merge `>>>` things here

Reply:
Or at least that's what they look like.

Reply:
This is the thing @indirectlylit mentioned today in the dev meeting: https://vue-loader.vuejs.org/en/features/scoped-css.html#deep-selectors

Reply:
This is the thing @indirectlylit mentioned today in the dev meeting: https://vue-loader.vuejs.org/en/features/scoped-css.html#deep-selectors

Reply:
the alternative syntax might be nicer if it works for us: `/deep/`

I like that it's an english word, and it doesn't look like CSS or a botched git merge conflict